android/quest/build.gradle.kts

```
@@ -512,6 +512,15 @@ dependencies {
   debugImplementation(libs.fragment.testing)
   // debugImplementation(libs.leakcanary.android)
 
+  kapt(libs.androidx.room.compiler)
+  implementation(libs.androidx.room.ktx)
+  implementation(libs.androidx.room.runtime)
+  implementation(libs.androidx.room.testing)
```

```suggestion
  testImplementation(libs.androidx.room.testing)
```

This should be a test dependency.